src/ripple/app/main/Main.cpp

```diff
@@ -49,6 +49,7 @@
 #include <cstdlib>
 #include <iostream>
 #include <utility>
+#include <stdexcept>
```

Comment:
You'll get different opinions on this one.  At a different company I worked on a refactoring project moving a bunch of C++ sources to a new directory tree to try and build a library.  A major component of the work was minimizing include dependencies between files.

Realistically, system includes never caused a problem.  System includes will be available anywhere.  But we found tons of non-system includes that were unneeded.  Those unneeded includes created artificial dependencies between files that were sometime a lot of work to break.

Since that time I've been inclined to minimize the includes in any source file, with particular emphasis on minimizing includes in *.h files.  If I can remove an include from a *.h file by putting that include in several *.cpp files, that's still a significant win.

Anyway that's my take on it.  Removing this system include is not necessarily important.  But, in general, I believe there's benefit in minimizing includes.